packages/trip-details/src/index.js

```diff
@@ -72,33 +75,39 @@ export default function TripDetails({
 
   return (
     <Styled.TripDetails className={className}>
-      <Styled.TripDetailsHeader>Trip Details</Styled.TripDetailsHeader>
+      <Styled.TripDetailsHeader>{messages.title}</Styled.TripDetailsHeader>
       <Styled.TripDetailsBody>
         <TripDetail
+          description={messages.departDescription}
           icon={<CalendarAlt size={17} />}
           summary={
             <Styled.Timing>
               <span>
-                Depart <b>{date.format(longDateFormat)}</b>
+                {messages.depart} <b>{date.format(longDateFormat)}</b>
               </span>
               {routingType === "ITINERARY" && (
                 <span>
                   {" "}
-                  at <b>{formatTime(itinerary.startTime, timeOptions)}</b>
+                  {messages.at}{" "}
```

Comment:
(In the future, the entire sentence should be extracted - the order of words changes from one language to another.)